docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
 and events from namespaces of degraded cluster operators. The collected
 information includes:
 
 - Definitions for non-running (terminated, pending) Pods
 - Previous (if container was terminated) and current logs of all related pod containers
 - Namespace events
 
-  - Location of pod definitions: config/pod/{namespace}/{pod}.json
-  - Location of pod container current logs:
-    config/pod/{namespace}/logs/{pod}/{container}_current.log
-  - Location of pod container previous logs:
-    config/pod/{namespace}/logs/{pod}/{container}_previous.log
-  - Location of events in archive: events/
-  - Id in config: clusterconfig/operators_pods_and_events
-  - Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/pod](./insights-archive-sample/pod)
+- [docs/insights-archive-sample/events](./insights-archive-sample/events)
+
+### Location in archive
+- `config/pod/{namespace}/{pod}.json`
+- `events/`
+- `config/pod/{namespace}/logs/{pod}/{container}_{current|previous}.log`
+
+### Config ID
+`clusterconfig/operators_pods_and_events`
+
+### Released version
+- 4.8.2
+
+### Backported versions
+- 4.6.35+
+- 4.7.11+
+
+### Changes
+- This gather used to be part of [ClusterOperators](#ClusterOperators) before version `4.8.2`.
+- The collected data was previously included as specifications for `ClusterOperators`, and it was initially
+introduced in version `4.3.0` and later backported to version `4.2.10+`.
 
 
 ## ClusterOperators
 
-collects all the ClusterOperators definitions and their related resources
+Collects all the ClusterOperators definitions and their related resources
 from the `operator.openshift.io` group.
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
+### API Reference
+- https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
 
-* Location of operators related resources: config/clusteroperator/{group}/{kind}/{name}
-* Location of operators in archive: config/clusteroperator/
-* Location of operators related resources in older versions: config/clusteroperator/{kind}-{name}
-* See: docs/insights-archive-sample/config/clusteroperator
-* Id in config: clusterconfig/operators
-* Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### Sample data
+- [docs/insights-archive-sample/config/clusteroperator](./insights-archive-sample/config/clusteroperator)
+
+### Location in archive
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| >= 4.2.0  | config/clusteroperator/ 									|
```

Comment:
This doesn't tell what the structure of the folder was. It is either redundant (the remaining lines cover all the locations across versions) or it is incomplete. I guess it is the former.

Reply:
This line should not exist. It should be just the one with `< 4.7.0`. Thank you!